src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -485,7 +485,7 @@
        *
        {@comments(util.getDocLines(context.paramComments(service, method)))}
        * @@return {Stream}
-       *   An object stream which emits {@context.typeDocument(methodConfig.getPageStreaming.getResourcesField.getType)} on 'data' event.
+       *   An object stream which emits {@apiMethodView.doc.pageStreamingResourceTypeName} on 'data' event.
```

Comment:
Why not move the whole return doc comment to `returnsDocLines` and get rid of `pageStreamingResourceTypeName`?